packages/quantic/force-app/main/default/lwc/quanticFacetValue/quanticFacetValue.css

```diff
@@ -17,3 +17,56 @@ li {
 .facet__value-checkbox {
   min-width: unset;
 }
+
+.facet__value_new-checkbox{
+  width: 20px;
+  height: 20px;
+  vertical-align: top;
+  border: solid 1px #cccccc;
```

Comment:
Where did this color `#cccccc` come from?

Reply:
These styles aims to make div look and act a checkbox.  Will try to find suitable styling from [SLDS CSS classes](https://www.lightningdesignsystem.com/utilities/margin/).